rubin_sim/scheduler/utils/utils.py

```diff
@@ -148,19 +148,24 @@ def restore_scheduler(
         Scheduler object.
     observatory : rubin_sim.scheduler.observatory.Model_observatory
         The observaotry object
-    filename : str
-        The output sqlite dayabase to use
+    in_obs : np.array or str
+        Array of observation, or path to sqlite database to restore from
```

Comment:
Could you clarify here if you need observations as came out of the scheduler, or observations as converted to the opsim database style using SchemaConverter? (or if either would work and the restore_scheduler will test which version it gets)